src/arch/armv8/armv8-r/inc/arch/mem.h

```diff
@@ -52,7 +52,7 @@ typedef union {
 
```

Comment:
Commit 4643b407738578911f02967ded07e2367e8c33ee description:
(-Werror=old-style-declaration) Additionalloy,
the 'const' keyword is not need in static inline functions.

Not sure how the info in parenthesis relates to the rest.
'Additionalloy' is not correctly spelled

Reply:
Fixed in commit d17aa5ff754f3798b49971a04d47421931ad43c7